core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -483,12 +484,19 @@ public static boolean wantIGVBAMLinking() {
 		String[] studies = igvBamLinkingStudies.split(":");
 		return (studies.length > 0) ? Arrays.asList(studies) : Collections.<String>emptyList();
 	}
-
-    // CHANGES TO THIS LIST MUST BE PROPAGATED TO 'CacheMapUtil'
+    
+	/** 
+     * @deprecated
+     * This function is replaced by the {@link PortalSecurityConfig#userAuthorizationEnabled()}
+     * method in the org.cbioportal.utils package.
+     * 
+     * When updating logic make sure to also update {@link PortalSecurityConfig#userAuthorizationEnabled()}.
+     */
+    @Deprecated
```

Comment:
@n1zea144 Is it correct that the _org.mskcc.cbio.portal.util_ lib will be removed in the future? Does it make sense to mark it as @deprecated?  I started to create functions that have identical behavior in a new _utils_ module in package `org.cbioportal.utils`. Do you agree with migration of (some of) the GlobalProperties logic elsewhere?

Reply:
The main goal is to decompose the core module into more transparent "top-level" modules and consolidate code when possible (like the model package in core with the top level model package used by the web service).  If you've created functions with identical behavior in the new utils module, I think its ok to deprecated these.  Will this PR or a follow-up replace calls to these methods with the new ones?

Reply:
@n1zea144 I am not looking for a replacement of the entire GlobalProperties class. I merely would like to clarify to new developers that functionality that is present in the _org.mskcc.cbio.portal_ package should not be used in _org.cbioportal_ packages. Revisiting this topic I think it is better not to add the @Deprecased annotation since it is not actually replaced.